sacro/templates/index.html

```diff
@@ -139,7 +139,7 @@ <h1 class="text-xl font-semibold leading-7 text-gray-900" data-sacro-el="outputT
   </main>
 
   {% #modal id="submitModal" title="Release and download" subtitle='<span class="count"></span> files are ready to be released' %}
-    <form action="{{ submission_url }}" method="post" id="approveForm" class="flex flex-col gap-3">
+    <form action="{{ create_url }}" method="post" id="approveForm" class="flex flex-col gap-3">
```

Comment:
`create_url` sounds a bit vague. What are we creating? Is it possible to be more specific here?

Reply:
Reviews… the only thing we create in the entire Django app 😂